doc/guide/guide-basics.rst

```diff
@@ -57,7 +57,7 @@ To begin, let us create a blank ``Qobj``:
 
 where we see the blank ``Qobj`` object with dimensions, shape, and data. Here the data corresponds to a 1x1-dimensional matrix consisting of a single zero entry.
 
-.. Hint:: By convention, the names of Python classes, such as ``Qobj()``, are capitalized whereas the names of functions are not.
+.. Hint:: By convention, Class objects in Python such as ``Qobj()`` differ from functions in the use of a beginning capital letter.
```

Comment:
This is much worse -- the original made it clear that it is the names that are capitalized, rather than the classes or functions themselves, and scanned much better.